src/rules/tsUtils.ts

```diff
@@ -1,287 +1 @@
-      node.callee.property.type === AST_NODE_TYPES.Identifier &&
-      TestCaseProperty.hasOwnProperty(node.callee.property.name))
-  );
-};
-
-export const isDescribe = (
-  node: TSESTree.CallExpression,
-): node is JestFunctionCallExpression<DescribeAlias> => {
-  return (
-    (node.callee.type === AST_NODE_TYPES.Identifier &&
-      DescribeAlias.hasOwnProperty(node.callee.name)) ||
-    (node.callee.type === AST_NODE_TYPES.MemberExpression &&
-      node.callee.object.type === AST_NODE_TYPES.Identifier &&
-      DescribeAlias.hasOwnProperty(node.callee.object.name) &&
-      node.callee.property.type === AST_NODE_TYPES.Identifier &&
-      DescribeProperty.hasOwnProperty(node.callee.property.name))
-  );
-};
-
-export const isLiteralNode = (node: {
-  type: AST_NODE_TYPES;
-}): node is TSESTree.Literal => node.type === AST_NODE_TYPES.Literal;
-
-export interface StringLiteral extends TSESTree.Literal {
-  value: string;
-}
-
-export type StringNode = StringLiteral | TSESTree.TemplateLiteral;
-
-export const isStringLiteral = (node: TSESTree.Node): node is StringLiteral =>
-  node.type === AST_NODE_TYPES.Literal && typeof node.value === 'string';
-
-export const isTemplateLiteral = (
-  node: TSESTree.Node,
-): node is TSESTree.TemplateLiteral =>
-  node && node.type === AST_NODE_TYPES.TemplateLiteral;
-
-export const isStringNode = (
-  node: TSESTree.Node | undefined,
-): node is StringNode =>
-  node !== undefined && (isStringLiteral(node) || isTemplateLiteral(node));
-
-export const hasExpressions = (
-  node: TSESTree.Node,
-): node is TSESTree.Expression =>
-  'expressions' in node && node.expressions.length > 0;
-
-/* istanbul ignore next we'll need this later */
-export const getStringValue = (arg: StringNode): string =>
-  isTemplateLiteral(arg) ? arg.quasis[0].value.raw : arg.value;
-
-const collectReferences = (scope: TSESLint.Scope.Scope) => {
-  const locals = new Set();
-  const unresolved = new Set();
-
-  let currentScope: TSESLint.Scope.Scope | null = scope;
-
-  while (currentScope !== null) {
-    for (const ref of currentScope.variables) {
-      const isReferenceDefined = ref.defs.some(def => {
-        return def.type !== 'ImplicitGlobalVariable';
-      });
-
-      if (isReferenceDefined) {
-        locals.add(ref.name);
-      }
-    }
-
-    for (const ref of currentScope.through) {
-      unresolved.add(ref.identifier.name);
-    }
-
-    currentScope = currentScope.upper;
-  }
-
-  return { locals, unresolved };
-};
-
-export const scopeHasLocalReference = (
-  scope: TSESLint.Scope.Scope,
-  referenceName: string,
-) => {
-  const references = collectReferences(scope);
-  return (
-    // referenceName was found as a local variable or function declaration.
-    references.locals.has(referenceName) ||
-    // referenceName was not found as an unresolved reference,
-    // meaning it is likely not an implicit global reference.
-    !references.unresolved.has(referenceName)
-  );
-};
+export * from './utils';
```

Comment:
is this some sort of todo?

Reply:
No, that was b/c of the merging of #364, which didn't have the renaming commit, and so was trying to import from `tsUtils`.

By having this, it means ts code merged into master won't immediately break when it's pulled into this branch by importing from `tsUtils` instead of `utils`.

Hindsight is 20/20: I shouldn't have renamed. I've actually now rebased the branch to remove the renaming commit, so this is gone :)